docs/docsite/rst/getting_started/index.rst

```diff
@@ -12,11 +42,11 @@ Control node
    A system on which Ansible is installed.
    You run Ansible commands such as ``ansible`` or ``ansible-inventory`` on a control node.
 
-Managed node
-   A remote system, or host, that Ansible controls.
+Configuration file
+   The ``ansible.cfg`` configuration file enables you to customize and set various aspects of Ansible behavior on the control node.
 
 Inventory
-   A list of managed nodes that are logically organized.
+   A list of remote systems (managed nodes) that are logically organized and which Ansible brings to the state that you need.
```

Comment:
This plus the `ansible.cfg` should go in a separate PR because the changes are outside the scope of introduction.

Reply:
But having a configuration is a main component you need to have in order for your automation to work, so at least a mention of config file should be there.

The separate PR could discuss ansible.cfg more indepth.

Reply:
That is still outside the scope of this PR, which is to add the intro section to getting started.